softlayer/vm/softlayer_virtual_guest_creator.go

```diff
@@ -49,7 +49,7 @@ func (c *softLayerVirtualGuestCreator) Create(agentID string, stemcell bslcstem.
 				return c.createByOSReload(agentID, stemcell, cloudProps, networks, env)
 			}
 		case "manual":
-			return nil, bosherr.Error("Support manual netowrk soon...")
```

Comment:
How does this work when you have a dynamic and a manual (which is absolutely required for SL)?